cargo-dist/src/config.rs

```diff
@@ -272,6 +272,10 @@ pub struct DistMetadata {
     /// Hosting provider
     #[serde(skip_serializing_if = "Option::is_none")]
     pub hosting: Option<Vec<HostingStyle>>,
+
+    /// Custom GitHub runners, mapped by triple
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub custom_runners: Option<HashMap<String, String>>,
```

Comment:
Do you mind renaming this field to `github_custom_runners`? We're planning to add support for other CI backends in the future, so we'll want to be able to distinguish between GitHub runners and other systems' runners.

Alternately, we could make `custom-runners` a struct where `github` is a top-level key, inside of which is a hash like the one you've defined here.

Reply:
Renamed to `github_custom_runners`.